parsigs/parse_sig_api.py

```diff
@@ -61,6 +60,13 @@ class StructuredSig:
 inflect_engine = inflect.engine()
 
 
+def read_json_file():  # add parameters path, name
```

Comment:
this function should not be public change the name to `_read_json_file` or create new module that responsible for files general operations (think about indicative name)

Reply:
this function can read only one json file the value that stores in `file_path1`

Reply:
if you will need in the future to use multiple json files in the program you will need to create the same function again. instead you need to create function with this signiture: `def read_json_file(path: str)-> None:`